jujugui/static/gui/src/app/views/topology/service.js

```diff
@@ -1378,11 +1394,6 @@ YUI.add('juju-topology-service', function(Y) {
           });
         }
       }
-      // Find the centroid of our hull of services and inform the
-      // topology.
-      if (vertices.length) {
-        this.findCentroid(vertices);
```

Comment:
Do we ever want to pan to a service when it gets updated? This fixes the issue where services were panned to during deploy.